docs/api-reference/next.config.js/exportPathMap.md

```diff
@@ -4,6 +4,8 @@ description: Customize the pages that will be exported as HTML files when using
 
 # exportPathMap
 
+`exportPathMap` allows you to specify a mapping of request paths to page destinations, to be used during export.
```

Comment:
```suggestion
`exportPathMap` allows you to specify a mapping of request paths to page destinations, only used by `next export`.
```

Reply:
Also can you move this to be under the example, not on top of the page 🙏 

All of our pages show the examples first. That may change in the future but for the moment it should keep being consistent.